csharp_dotnetcore/14.NLP-With-Dispatch/README.md

```diff
@@ -0,0 +1,146 @@
+﻿This sample shows how to integrate QnA Maker in a simple bot with ASP.Net Core 2 and Application Insights. 
+
+# Concepts introduced in this sample
+
+[Dispatch](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Dispatch) is a tool to create and evaluate LUIS models used for NLP (Natural Language Processing). Dispatch works across multiple bot modules such as LUIS applications, QnA knowledge bases and other NLP sources (added to dispatch as a file type). Use the Dispatch model in cases when:
+* Your bot consists of multiple modules and you need assistance in routing user's utterances to these modules and evaluate the bot integration.
+* Evaluate quality of intents classification of a single LUIS model.
+* Create a text classification model from text files.
+
+The [Language Understanding Intelligent Service (Luis)](https://www.luis.ai), is a machine learning-based service to build natural language into apps, bots, and IoT devices. Luis allows to Quickly create enterprise-ready, custom models that continuously improve. 
+
+The [QnA Maker Service](https://www.qnamaker.ai) enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes.
+
+The [Application Insights](https://azure.microsoft.com/en-us/services/application-insights/) enables you to discover actionable insights through application performance management and instant analytics.
+
+In this sample, we demonstrate how to use the Dispatch service to route utterances when there are multiple Luis models and QnA maker services for different scenarios supported by a bot. In this case, we confiure dispatch with multiple Luis models for conversations around home automation and weather information, plus QnA Maker service to answer questions based on a FAQ text file as input.
+
+# To try this sample
+
+- Clone the samples repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+## Install BotBuilder tools
+
+- In a terminal, navigate to the samples folder (`BotBuilder-Samples\csharp_dotnetcore\14.NlpWithDispatch`) 
+
+    ```bash
+    cd BotBuilder-Samples\csharp_dotnetcore\14.NlpWithDispatch
+    ```
+
+- Install required tools - to successfully setup and configure all services this bot depend on, you need to install the MSBOT, LUIS, QnAMaker, Ludown, Dispatch CLI tools. 
+    ```bash
+    npm i -g msbot luis-apis qnamaker ludown botdispatch
+    ```
+- Configure required LUIS, QnA Maker and Dispatch services. See [here](#configure-services)
```

Comment:
>LUIS [](start = 21, length = 4)

Casing here is LUIS, (and in other places). I think the text above (in the previous sections) should follow the same casing.